lib/vsc/utils/optcomplete.py

```diff
@@ -617,19 +618,19 @@ def gen_cmdline(cmd_list, partial, shebang=True):
     @param partial: the string to autocomplete (typically, partial is an element of the cmd_list)
     @param shebang: script has python shebang (if not, add sys.executable)
     """
-    cmdline = " ".join(cmd_list)
+    cmdline = ' '.join(['"%s"' % cmd for cmd in cmd_list])
```

Comment:
I know why you did this, but this may look weird when we run into this again later; add a comment why you're injecting double quotes around the commands?

Also, what if the command includes double quotes already?